korangar_util/src/loader.rs

```diff
@@ -10,6 +9,12 @@ impl FileNotFoundError {
     }
 }
 
+impl std::fmt::Debug for FileNotFoundError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Can't find file: {}", self.0)
```

Comment:
Actually super small nit: I am not super consisent with this myself but it is best practice to start error messages (and panic messages) with a lowercase character (and end without a trailing dot)

Reply:
Fixed